store/sqlstore/store.go

```diff
@@ -233,7 +240,12 @@ func New(settings model.SqlSettings, metrics einterfaces.MetricsInterface) *SqlS
 	store.stores.linkMetadata.(*SqlLinkMetadataStore).createIndexesIfNotExists()
 	store.stores.group.(*SqlGroupStore).createIndexesIfNotExists()
 	store.stores.scheme.(*SqlSchemeStore).createIndexesIfNotExists()
-	store.stores.preference.(*SqlPreferenceStore).deleteUnusedFeatures()
+
+	store.asyncMigrations = append(store.asyncMigrations, store.stores.preference.(*SqlPreferenceStore).generateAsyncMigrations()...)
```

Comment:
Wondering if it makes sense to keep the the migrations high-level vs. anchoring them to a single store? This makes sense for things like indices, but I suspect some of the data migrations we've already written (and ones to come) will span stores, and it might be awkward to have them scattered.

Reply:
Totally with you on this, i've just followed what we follow for the sync indexes. We basically call `createIndexesIfNotExists` for each of the stores. Happy to keep it in one place, 0/5 here